src/components/ParticipantList.jsx

```diff
@@ -27,13 +29,16 @@ const timeAgo = new TimeAgo('en-US')
 // TODO: Traffic Lights with Last Survey Date + Login+device + # completed events
 
 
-export default function ParticipantList({ studyID, title, onParticipantSelect, showUnscheduled, ...props }) {
+export default function ParticipantList({ studyID, title, onParticipantSelect, showUnscheduled, credential, ...props }) {
```

Comment:
It was needed for the qr code function to work

Reply:
You can get rid of it if you make the changes I suggested below. Right now the issue with this is we're not passing in the `credential` prop so it'll always be `undefined`. If you take a look then at the `credID` variable, it'll always default to `''` (empty string). Furthermore, the `password` state variable is always set to `''` (empty string)` so that QR code that's being generated is only getting empty strings. If someone tried to log in using that QR code it would actually fail or crash.

Reply:
removed